src/SQLClient/index.js

```diff
@@ -0,0 +1,17 @@
+const knex = require('knex');
```

Comment:
I was reading that calling fiiles `index.js` is not a good idea because it's too generic and makes it harder to search files by name. I suggest to call this file `db.js` or `client.js`